dev/com.ibm.ws.jbatch.jms/src/com/ibm/ws/jbatch/jms/internal/listener/BatchJmsEndpointListener.java

```diff
@@ -569,10 +577,15 @@ private void handleStartRequest(long instanceId, long executionId, byte[] securi
                 return;
             }
             
-            WSJobInstance jobInstance = jobRepositoryProxy.updateJobInstanceStateOnConsumed(instanceId);
-            if (jobInstance.getInstanceState() != InstanceState.JMS_CONSUMED) {
+            WSJobInstance jobInstance = null; 
+
+            try {
+            	jobInstance = jobRepositoryProxy.updateJobInstanceStateOnConsumed(instanceId);
+            } catch (JobInstanceNotQueuedException exc) {
+            	// trace and swallow this 
                 if(tc.isDebugEnabled()) {
-                    Tr.debug(BatchJmsEndpointListener.this, tc, "Exiting since instanceState isn't equal to JMS_CONSUMED, instead is: " + jobInstance.getInstanceState());
+                	// Might be misleading to trace the non-JMS_CONSUMED `state we found without locking it down
```

Comment:
It'd be better to change "JMS_CONSUMED" here to "JMS_QUEUED", even though it's just a comment (it could help someone later to be more clear).

When we wrote this before, we were looking for a _return value_ with state of **JMS_CONSUMED**.   Now that we more simply throw an exception to indicate the wrong (non-JMS_QUEUED) state, it'd be clearer to mention the issue is that we didn't find this in the state of **JMS_QUEUED**

Now that we've simplified the